src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -58,49 +61,51 @@ object FireCloudConfig {
 
   object Thurloe {
     private val profile = config.getConfig("userprofile")
-    lazy val baseUrl = sys.env.get("THURLOE_URL_ROOT").get
-    lazy val authPrefix = profile.getString("authPrefix")
-    lazy val authUrl = baseUrl + authPrefix
-    lazy val setKey = profile.getString("setKey")
-    lazy val get = profile.getString("get")
-    lazy val getAll = profile.getString("getAll")
-    lazy val getQuery = profile.getString("getQuery")
-    lazy val delete = profile.getString("delete")
-    lazy val postNotify = profile.getString("postNotify")
+    private val thurloe = config.getConfig("thurloe")
+    val baseUrl = thurloe.getString("baseUrl")
+    val authPrefix = profile.getString("authPrefix")
+    val authUrl = baseUrl + authPrefix
+    val setKey = profile.getString("setKey")
+    val get = profile.getString("get")
+    val getAll = profile.getString("getAll")
+    val getQuery = profile.getString("getQuery")
+    val delete = profile.getString("delete")
+    val postNotify = profile.getString("postNotify")
```

Comment:
Something to consider (not for this change): Is there any way we can get rid of some of this boilerplate?